### PR TITLE
Show resource usage data.

### PR DESCRIPTION
**Pull request by porridge**

### CODE REVIEW

**Comment by porridge** [comment]
Bouncing the PR to kick Travis.

File: perfdash/parser.go [hunk 1]
```diff
@@ -43,6 +43,52 @@ func parseTestOutput(data []byte, buildNumber int, job string, testName string,
...
+		result[testName] = BuildData{Job: job, Version: "v1", Builds: map[string][]perftype.DataItem{}}
+	}
+	usage := make(podNameToUsage)
+	for percentile, items := range obj {
+		for _, item := range items {
+			name := RemoveDisambiguationInfixes(item.Name)
+			if _, ok := usage[name]; !ok {
+				usage[name] = make(usageAtPercentiles)
+			} // TODO(porridge): keep max usage?
+			usage[name][percentile] = resourceUsage{float64(item.Cpu), float64(item.Memory)}
```
Comment (gmarek) [inline perfdash/parser.go]: You keep random one, right?

Comment (porridge) [reply perfdash/parser.go]: Yes, effectively the last one spit out by the test to the `.json` file.

Comment (gmarek) [reply perfdash/parser.go]: Max is probably more interesting as it should be more stable than a random one.

Comment (porridge) [reply perfdash/parser.go]: Right, but max over which dimension? :-/

Comment (porridge) [reply perfdash/parser.go]: Or should we keep a franken-record of `{max(cpu), max(memory)}` rather than `max({cpu,memory})`?

Comment (gmarek) [reply perfdash/parser.go]: Max(cpu), max(memory) - this one should be +- stable.

Comment (porridge) [reply perfdash/parser.go]: Done, PTAL.

**Pull request merged**